muselsl/muse.py

```diff
@@ -40,72 +42,52 @@ def __init__(self, address=None, callback_eeg=None, callback_control=None,
         self.interface = interface
         self.time_func = time_func
 
-        if backend in ['auto', 'gatt', 'bgapi']:
-            if backend == 'auto':
-                if platform == "linux" or platform == "linux2":
-                    self.backend = 'gatt'
-                else:
-                    self.backend = 'bgapi'
-            else:
-                self.backend = backend
-        else:
-            raise(ValueError('Backend must be auto, gatt or bgapi'))
+        self.backend = helper.resolve_backend(backend)
 
     def connect(self, interface=None, backend='auto'):
         """Connect to the device"""
+        try:
+            if self.backend == 'bluemuse':
+                print('Starting BlueMuse.')
+                subprocess.call('start bluemuse:', shell=True)
 
-        if self.backend == 'gatt':
-            self.interface = self.interface or 'hci0'
-            self.adapter = pygatt.GATTToolBackend(self.interface)
-        else:
-            self.adapter = pygatt.BGAPIBackend(serial_port=self.interface)
-
-        self.adapter.start()
-
-        if self.address is None:
-            address = self.find_muse_address(self.name)
-            if address is None:
-                raise(ValueError("Can't find Muse Device"))
             else:
```

Comment:
any reason to split else if statement instead of elif below?

Reply:
Probably not. I'm just not used to Elif coming from JS

Reply:
Just took a look at this. I think the way it's written now is best. BlueMuse is very different from the other backends and doesn't use many of the functions the others do. Because of that, keeping everything else in it's own else bloack is the most readable